.circleci/config.yml

```diff
@@ -45,57 +44,101 @@ aliases:
     steps:
       - *restore_git_cache
       - checkout
-      - *restore_npm_cache
       - run:
-          name: "Run npm test to build"
+          name: "setup"
+          command: |
+            npm --production=false ci
+            mkdir ./test/results
+      - run:
+          name: "run lint tests"
+          command: |
+            npm run test:lint:ci
+      - run:
+          name: "run npm build"
           command: |
-            npm --production=false install
-            WWW_VERSION=${CIRCLE_SHA1:0:5} npm run test
+            WWW_VERSION=${CIRCLE_SHA1:0:5} npm run build
+      - run:
+          name: "Run unit tests"
+          command: |
+            npm run test:unit:jest
+            npm run test:unit:tap:ci
+            npm run test:unit:convertReportToXunit
       - *save_npm_cache
       - *save_git_cache
       - *save_build_cache
+      - store_test_results:
+          path: test/results
+      - store_artifacts:
+          path: test/results
```

Comment:
We want this to be the build directory. And since we will have uploaded the build at this point, we won't need `store_artifacts` in any other job.

Reply:
I removed these lines